glassfish-runner/jdbc-platform-tck/pom.xml

```diff
@@ -0,0 +1,548 @@
+                                <argument>--maxpoolsize</argument>
+                                <argument>${jdbc.maxpoolsize}</argument>
+                                <argument>${poolName}</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>5-create-jndi</id>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <phase>pre-integration-test</phase>
+                        <configuration>
+                            <executable>${exec.asadmin}</executable>
+                            <arguments>
+                                <argument>create-jdbc-resource</argument>
+                                <argument>--connectionpoolid</argument>
+                                <argument>${poolName}</argument>
+                                <argument>${jndiName}</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>6-list-jdbc-connection-pools</id>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <phase>pre-integration-test</phase>
+                        <configuration>
+                            <executable>${exec.asadmin}</executable>
+                            <arguments>
+                                <argument>list-jdbc-connection-pools</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>7-list-jdbc-resources</id>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <phase>pre-integration-test</phase>
+                        <configuration>
+                            <executable>${exec.asadmin}</executable>
+                            <arguments>
+                                <argument>list-jdbc-resources</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>8-stop-domain</id>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <phase>pre-integration-test</phase>
+                        <configuration>
+                            <executable>${exec.asadmin}</executable>
+                            <arguments>
+                                <argument>stop-domain</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
```

Comment:
execution tasks 1-stop-domain to 8-stop-domain can be moved to glassfishbootCommand like in https://github.com/jakartaee/platform-tck/blob/ad8ce5ae2cbc945ddb84f8fa3cd1b379e5023115/glassfish-runner/jta-platform-tck/pom.xml#L383 .